accepted/0032-scene-wizard.md

```diff
@@ -0,0 +1,36 @@
+- Start Date: 2020-09-10
+- RFC PR: #32
+
+# Summary
+
+This proposal is for adding a scene wizard for new users when first loading OBS.
+
+# Motivation
+
+Users often find it difficult to set up OBS.
+
+# Detailed Design
+
+The user will also be able to open the wizard in the scene collection menu.
+
+### Wizard page 1
+- Buttons that allow the user to select how they want to use OBS
+	1. "I just want to record my screen"
+	2. "I want to use a scene template"
+	3. "I will set up OBS myself" (if they select this then exit wizard)
+
+- The scene templates would be just pre-made scene collections
+
+### Wizard page 2
+- If the user selects scene template, show them different scene collections they could choose from
+- The scene collections will have image previews as well
+- If the user selects screen recording, ask them what monitor they want to record
+
+### Wizard page 3
+- Have the user select their microphone and webcam
+- The webcam would have a preview
+- The microphone would also show a volume meter
```

Comment:
I'd personally vote that this should be where we include other basic things like choosing default microphone filters (ideally, at least, a noise suppression filter & a compressor). We should also make it super clear to the user if their mic is peaking.

Speaking of peaking, I wonder if it'd be possible to determine what a device's system volume is in dB. Windows does have the UI for it, and being able to notify a user that their mic is "above the recommended range" (ie above 0dB). 
![image](https://user-images.githubusercontent.com/941350/93014185-5084e980-f5f2-11ea-99de-b243f12de543.png)

Reply:
One easy way to include the filters, is to have them in the scene template, which is just a scene collection.